__tests__/hooks/isAuthenticated.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { renderHook } from '@testing-library/react-hooks';
+import fetchMock from 'jest-fetch-mock';
+import IsAuthenticated from '@/hooks/isAuthenticated';
+
+beforeAll(() => {
+    fetchMock.enableMocks();
+});
+
+afterEach(() => {
+    fetchMock.resetMocks();
+});
+
+it('should return isLoggedIn as true and userData if the request is successful', async () => {
+    const userDataMock = {
+        data: {
+            Id: 1,
+            Username: 'Sunny Sahsi',
+            Email: 'sunnysahsi@gmail.com',
+            Password: '',
+            IsVerified: false,
+            IsOnboarding: true,
+            CreatedAt: '2023-11-04T10:02:26.582699Z',
+            UpdatedAt: '2023-11-04T10:02:26.582699Z',
+        },
+        message: 'user fetched successfully',
+    };
```

Comment:
Move this to fixtures

Reply:
For now we can keep in this file , in future if we have a large amount of data then we can create the fixtures directory

Reply:
You can do it now it not take too much time because in future we can add in that fixture file and test also easy to read